[CDS-96554] Added CrashLoopBackoff troubleshooting (#8084)
harness/developer-hub: docs/continuous-delivery/gitops/troubleshooting.md

- To manage rate limiting in GitHub, see [Enabling rate limits for Git](https://docs.github.com/en/enterprise-server@3.10/admin/configuration/configuring-user-applications-for-your-enterprise/configuring-rate-limits#enabling-rate-limits-for-git).

### Issue: GitOps agent pod stuck in CrashLoopBackoff

> The GitOps agent goes into a CrashLoopBackoff state due to an authentication failure with the GitOps service. This issue occurred as the token in the agent’s YAML did not match the public key in the GitOps service database.

Reviewer: ```suggestion
The GitOps agent goes into a CrashLoopBackoff state due to an authentication failure with the GitOps service. This issue occurs when the token in the agent’s YAML does not match the public key in the GitOps service database.
```
